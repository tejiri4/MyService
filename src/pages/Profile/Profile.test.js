import React from "react";
import { shallow } from "enzyme";
import ProfilePage from "./index";

describe("Profile Page", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ProfilePage />);
  });

  it("renders the profile page component without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});

