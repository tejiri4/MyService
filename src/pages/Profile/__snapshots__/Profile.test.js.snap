// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Page renders the profile page component without crashing 1`] = `ShallowWrapper {}`;
